src/pages/reportSighting/ReportForm.jsx

```diff
@@ -60,6 +60,27 @@ export default function ReportForm({
     data: siteSettingsData,
     siteSettingsVersion,
   } = useSiteSettings();
+  const recaptchaPublicKey = get(siteSettingsData, [
```

Comment:
This is just a merge I did with the recaptcha branch so that I could submit AGSs.

Reply:
not really best practice but i will ignore :)